app/assets/javascripts/pretty-text/engines/discourse-markdown/image-controls.js

```diff
@@ -79,37 +119,21 @@ export function setup(helper) {
       "span.separator",
       "span.scale-btn[data-scale]",
       "span.button-wrapper[data-image-index]",
+      "span[aria-label]",
+      "span.alt-text-readonly-container",
+      "span.alt-text-readonly-container.alt-text",
+      "span.alt-text-readonly-container.alt-text-edit-btn",
+      "svg[class=fa d-icon d-icon-pencil svg-icon svg-string]",
+      "use[xlink:href=#pencil-alt]",
+      "input[type=text]",
+      "input[hidden=true]",
+      "input[class=alt-text-input]",
```

Comment:
I'm not sure if there's a better way to whitelist these. I tried adding these in another file's `helper.allowList` and it seems it's universal. I'd feel better off if I could do something more precise like `input[class=alt-text-input][hidden=true]`.

Reply:
I think what you did here is good. Just a note we try to only use the term allowList now.

Reply:
> we try to only use the term allowList now

Do you mean `allowList` is temporary? Can you elaborate please?

Reply:
Ahh sorry, a year or 2 ago we made a policy that in all places where we had `abcWhitelist` and `abcBlacklist` instead we use `abcAllowList` and `abcDenyList`, slightly less loaded terms.